app/admin-dashboard/[slug]/page.tsx

```diff
@@ -0,0 +1,12 @@
+'use client';
+
+import { useSearchParams } from 'next/navigation';
+
+const UsersPage = () => {
+  const searchParams = useSearchParams();
+
+  const search = searchParams.get('id');
+  return <div>Users: {search}</div>;
+};
+
+export default UsersPage;
```

Comment:
Not sure why these changes are on this PR. Please revert the change as it is not applicable here.